packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -205,6 +205,32 @@ class WebViewController {
     return _channel.invokeMethod('loadUrl', url);
   }
 
+  /// Check that there is back item in the history list.
+  Future<bool> canGoBack() async {
+    final bool canGoBack = await _channel.invokeMethod("canGoBack");
+    return canGoBack;
+  }
+
+  /// Check that there is forward item in the history list.
+  Future<bool> canGoForward() async {
+    final bool canGoForward = await _channel.invokeMethod("canGoForward");
+    return canGoForward;
+  }
+
+  /// Navigates to the back item in the history list.
```

Comment:
Suggestions:

```dart
/// Goes back in the history of this WebView.
///
/// If there is no back history item this is a no-op.
```

Actually it seems like at least the Android WebView's documentation doesn't make any guarantees about what happens if we can't go back. Lets check if we can go back in our Android code before invoking `goBack` (left a comment there as well)

Reply:
> least the Android WebView's documentation doesn't make any guarantees about what happens if we can't go back

I also worry about it. Checking in d0e68de, and fix doc comment for it in 1df519b.